tekton/ui/pipeline.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright © 2021 The Tekton Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ui-pipeline
+spec:
+  params:
+    - name: HUB_REPO
+    - name: REVISION
+    - name: IMAGE
+    - name: TAG
+    - name: HUB_NAMESPACE
+    - name: K8S_VARIANT
+  workspaces:
+    - name: shared-workspace
+    - name: kubeconfig-dir
+      optional: true
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: $(params.HUB_REPO)
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
+          value: "true"
+        - name: revision
+          value: $(params.REVISION)
+    - name: install-dependencies
+      taskRef:
+        name: npm
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+      params:
+        - name: PATH_CONTEXT
+          value: "ui"
+        - name: ARGS
+          value:
+            - clean-install
+      runAfter:
+        - fetch-repository
+    - name: run-lint
+      taskRef:
+        name: npm
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+      params:
+        - name: PATH_CONTEXT
+          value: "ui"
+        - name: ARGS
+          value:
+            - run
+            - lint
+      runAfter:
+        - install-dependencies
+    - name: build-test
+      taskRef:
+        name: npm
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+      params:
+        - name: PATH_CONTEXT
+          value: "ui"
+        - name: ARGS
+          value:
+            - run
+            - build
+      runAfter:
+        - install-dependencies #run-lint
```

Comment:
Why we we have `#run-lint` as a comment ??

Reply:
fixed it